### PR TITLE
Accessory Shop & Kingdom credits

### PR DESCRIPTION
**Pull request by mstww**
I don't think there is a bug, it works fine, I tested it in many ways

### CODE REVIEW

File: discord/bot.js [hunk 1]
```diff
@@ -1129,6 +1132,10 @@ client.on("interactionCreate", async (interaction) => {
                     break;
                 }
                 case "valstatus": {
+                    if (!valorantUser) return await interaction.reply({
+                        embeds: [basicEmbed(s(interaction).error.NOT_REGISTERED)],
+                        ephemeral: true
+                    });
```
Comment (giorgi-o) [inline discord/bot.js:1138]: Is there a reason someone should need an account to use /valstatus?

File: discord/bot.js [hunk 2]
```diff
@@ -1447,15 +1453,18 @@ client.on("interactionCreate", async (interaction) => {
...
                         ephemeral: true
-                });
+                    });
+                }
 
                 let newMessage;
                 switch(customId) {
-                    case "shop": newMessage = await fetchShop(interaction, getUser(interaction.user.id)); break;
+                    case "shop": newMessage = await fetchShop(interaction, getUser(interaction.user.id), interaction.user.id, "d"); break;
+                    case "accessoryshop": newMessage = await fetchShop(interaction, getUser(interaction.user.id), interaction.user.id, "a"); break;
```
Comment (giorgi-o) [inline discord/bot.js:1467]: It wasn't immediately obvious that d = daily and a = accessory. Could you please either rename them to "daily" and "accessory", or make an enum somewhere (probably util.js), or just leave a comment in the `fetchShop()` definition that says something like `// "accessory" can be either "d" for daily, or "a" for accessory`

**Comment by giorgi-o** [comment]
Overall, looks great, and also fixes a few long-standing bugs and code quality stuff! Feel free to merge once you've addressed those two comments 👍

**Review by giorgi-o — APPROVED** [review]
Overall, looks great, and also fixes a few long-standing bugs and code quality stuff! Feel free to merge once you've addressed those two comments 👍

File: discord/bot.js (hunk 1 above)
Comment (Kyedae) [reply discord/bot.js:1138]: It's needed considering the command goes based off the users region, so if they don't have an account it will just not work.

**Pull request merged**